.eslintrc.js

```diff
@@ -0,0 +1,232 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier",
```

Comment:
Since we should prettier here, I think we can get rid of most of custom rules here. So we can keep the rules compact and simplistic.